src/Resources/require.js

```diff
@@ -132,8 +132,9 @@
     };
 
     window.register = function (name, initializer) {
-        if (_modules[name]) {
-            throw new ModuleRedeclareError(name);
+        if (_modules[name] && typeof console !== 'undefined' && typeof console.error === 'function') {
```

Comment:
This could be a separate PR, but
- The a test case at `spec/require.spec.js` should be extended to cover this case
- Right now there if there is a console it keeps the old version of the package, if there is no console it will get the new version.